### DISCUSSION TITLE
Header not included in copy

### DISCUSSION THREAD

**Discussion by grazielleanna** [nick-keller/react-datasheet-grid#129] [Feature requests]
The library does not allow copying the header. How to make this possible? Urgent!!

**Comment by nick-keller** [comment]
Hello @grazielleanna,
This is a free open-source project and delivery is therefore mostly driven by the time I am willing to put into the project. 
You can drastically speed up the process by contributing and sending a pull request!

Now regarding your feature request, it looks like you saw the related thread https://github.com/Equify/react-datasheet-grid/discussions/62
Happy to help if you are struggling to implement it, please share some code as a base for discussion.
